pkg/verifier/notation/notation.go

```diff
@@ -44,8 +44,9 @@ import (
 )
 
 const (
-	verifierName    = "notation"
-	defaultCertPath = "ratify-certs/notation/truststore"
+	verifierName       = "notation"
+	defaultCertPath    = "ratify-certs/notation/truststore"
+	namespaceSeperator = "/"
```

Comment:
nit: is this universal? should this also be moved to global constants?

Reply:
moved to internal const. thanks!